docs/source/sdks.rst

```diff
@@ -0,0 +1,22 @@
+=================
+Sawtooth SDK APIs
+=================
+
+- Python
+
+  - `Transaction Processor
+    <https://sawtooth.hyperledger.org/docs/core/nightly/master/python_sdk/processor/sawtooth_sdk.processor.html>`__
```

Comment:
Is there any way to make these relative links?

Reply:
I agree that relative links would be good.  Here's a possible way to do this (kinda hacky): https://stackoverflow.com/questions/27979803/external-relative-link-in-sphinx-toctree-directive